src/zkevm_specs/evm/execution/__init__.py

```diff
@@ -42,7 +43,7 @@
 from .extcodehash import *
 from .extcodesize import *
 from .log import *
-from .bitwise import not_opcode
+from .not_ import not_opcode
```

Comment:
Why the underscore at the end?
```suggestion
from .not import not_opcode
```

Reply:
Thank you for the review.

> Why the underscore at the end?

`not` is reserved word so we can't use it for module and function name.
We also had same situation before as in following.
https://github.com/privacy-scaling-explorations/zkevm-specs/blob/7b5e8b43e2aee0783e5cecb129465001fbc3d758/src/zkevm_specs/evm/execution/return_.py#L6

These are some ways to avoid this.
I avoid this by making one of them big character as following.
https://github.com/privacy-scaling-explorations/zkevm-specs/pull/352/files#diff-cf5d5239297dc4263160de42e8c29b203276c16d0451220732c9ef739ba4bc3bR37

Please let me know if you have any preference 😃
Thank you!

Reply:
Ohhh 😅😅😅

I should look a tiny bit more into Python lol